dftimewolf/lib/processors/turbinia.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+"""Processes cloud artifacts using a remote Turbinia instance."""
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+import os
+import tempfile
+
+from dftimewolf.lib.module import BaseModule
+
+from turbinia import client as turbinia_client
+from turbinia import config
+from turbinia import evidence
+from turbinia import output_manager
+from turbinia import TurbiniaException
+from turbinia.message import TurbiniaRequest
+
+
+class TurbiniaProcessor(BaseModule):
+  """Process cloud disks with remote Turbinia instance.
+
+  Attributes:
+    client: A TurbiniaClient object
+    disk_name: Name of the disk to process
+    instance (string): The name of the Turbinia instance
+    project: The project containing the disk to process
+    region (string): The region Turbinia is in
+    zone: The zone containing the disk to process
+    _output_path: The path to output files
```

Comment:
Since this one is private (i.e. leading `_`) I'm not sure it belongs in the docstring.

Reply:
I'll defer to you since I'm sure you want it to be consistent with the rest of the project, but I personally always prefer to keep the pseudo-private members as part of the docstrings so that they are still documented somewhere.

Reply:
SGTM